src/components/chapters/Info/InfoPage.tsx

```diff
@@ -14,7 +13,7 @@ interface Props {
 }
 const InfoPage: React.FC<Props> = ({ hasError, chapterInfoResponse, chapterResponse }) => {
   if (hasError) {
-    return <Error statusCode={500} />;
+    return <Error />;
```

Comment:
I feel like this status code will be important in the future, even if we don't make use of it now. We might be showing different content on the error page depending on the error.